komodo/cli.py

```
@@ -2,158 +2,206 @@
     # (release)
-    tmp_prefix = os.path.join(os.path.join(args.prefix), args.release, 'root')
+    tmp_prefix = os.path.join(os.path.join(args.prefix), args.release, "root")
     fakeroot = os.path.abspath(args.release)
     if args.build or not args.install:
-        komodo.make(args.pkgs, args.repo, prefix   = tmp_prefix,
-                                         dlprefix = args.cache,
-                                         builddir = args.tmp,
-                                         jobs     = args.jobs,
-                                         cmk      = args.cmake,
-                                         pip      = args.pip,
-                                         fakeroot = fakeroot)
-        komodo.shell('mv {} {}'.format(args.release + tmp_prefix, args.release))
-        komodo.shell('rmdir -p --ignore-fail-on-non-empty {}'.format(
-            os.path.dirname(args.release + tmp_prefix)))
+        komodo.make(
+            args.pkgs,
+            args.repo,
+            data,
+            prefix=tmp_prefix,
+            dlprefix=args.cache,
+            builddir=args.tmp,
+            jobs=args.jobs,
+            cmk=args.cmake,
+            pip=args.pip,
+            virtualenv=args.virtualenv,
+            fakeroot=fakeroot,
+        )
+        komodo.shell("mv {} {}".format(args.release + tmp_prefix, args.release))
+        komodo.shell(
+            "rmdir -p --ignore-fail-on-non-empty {}".format(
+                os.path.dirname(args.release + tmp_prefix)
+            )
+        )
 
     if args.build and not args.install:
         sys.exit(0)
 
     # create the enable script
-    for tmpl,target in [('enable.in','enable'), ('enable.csh.in', 'enable.csh')]:
-        if not os.path.exists(tmpl):
-            logging.warn('Could not find template %s, skipping.' % str(tmpl))
-            continue
+    for tmpl, target in [("enable.in", "enable"), ("enable.csh.in", "enable.csh")]:
+        # if not os.path.exists(tmpl):
```

either comment back in or remove

---

Right, we don't want to warn here, we want to fail hard. Will fix.